src/i18n/es-es/index.js

```diff
@@ -242,6 +242,7 @@ export default {
         to_interacted_with: 'A',
         value: 'Valor',
         txn_fee: 'Txn Fee',
+        txn_failed: 'Txn Fallido',
```

Comment:
In Spanish, we have gendered nouns. "La transacción" is feminine, but the adjective "fallido" is used to match the masculine noun, so you need to use "fallida".

```suggestion
        txn_failed: 'Txn Fallida',
```